db/migrate_cards/20150318162100_import_new_source_type_and_set_pattern.rb

```diff
@@ -0,0 +1,16 @@
+# -*- encoding : utf-8 -*-
+
+class ImportNewSourceTypeAndSetPattern < Card::Migration
+  def up
+    Card.create! :name=>"TextSource", :type_id=>Card::CardtypeID, :codename=>"text_source"
+    
+    Card.create! :name=>'Citable', :type_id=>Card::SetID, :codename=>'citable' 
+      ,:content=>%{
+        {
+          "type":["in","SourceFile","Page","TextSource"]
+        }
+      }
```

Comment:
Maybe it's time to rename 'Page' to 'WebSource'?  I think it should be consistent how we compose the names of those types. Either SourceFile and SourceText or FileSource and TextSource.